bulk-fhir-tests/src/test/java/gov/va/api/health/bulkfhir/tests/PublicationIT.java

```diff
@@ -20,52 +23,40 @@
 import lombok.NoArgsConstructor;
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
+import org.junit.After;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.mockserver.integration.ClientAndServer;
 
 @Slf4j
 public class PublicationIT {
 
+  @After
+  public void cleanUpPublications() {
```

Comment:
I think you should also do this `@Before` to make sure it's clean, just in case the previous test failed to clean up after itself.

Also, should we consider naming the publications something more obvious they are part of the integration tests, e.g. 
`PublicationIt.class.getSimpleName() + "-fullCycle-1"`
That way we ever see them in the wild, we know which test needs work.